p/api/greader.php

```diff
@@ -30,17 +30,21 @@
 
 if (PHP_INT_SIZE < 8) {	//32-bit
 	function hex2dec(string $hex): string {
-		if (!ctype_xdigit($hex)) return '0';
+		if (!ctype_xdigit($hex)) {
```

Comment:
Although I do not mind the proposed syntax, please avoid minor stylistic code changes, especially when not related to the PR.

Reply:
No need to change it back though

Reply:
done